packages/paste-website/src/components/DoDont.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+const StyledWrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  grid-column-gap: 24px;
+`;
+
+interface DoDontProps {
+  children?: React.ReactNode;
+}
+
+function DoDont(props: DoDontProps) {
+  return <StyledWrapper>{props.children}</StyledWrapper>;
+}
+
+const StyledChild = styled.div``;
+
+const StyledChildImg: React.FC<ChildProps> = styled.div`
+  margin-bottom: 16px;
+  border: 1px solid #ccd2dc;
+  border-bottom: ${(props: ChildProps) => (props.do ? '2px solid #23bf6e' : '2px solid #ce241a')};
+`;
```

Comment:
If you wanted to you could use tokens to style this.
```
const StyledChildImg: React.FC<ChildProps> = styled.div`
  margin-bottom: ${themeGet('space.space50')};
  border: ${themeGet('borderWidths.borderWidth10')} solid ${themeGet('borderColors.colorBorderLight')};
  border-bottom: ${(props: ChildProps) =>
    props.do
      ? `${themeGet('borderWidths.borderWidth20')(props)} solid ${themeGet('borderColors.colorBorderSuccess')(props)}`
      : `${themeGet('borderWidths.borderWidth20')(props)} solid ${themeGet('borderColors.colorBorderError')(props)}`};
`;```

The Box component may not be ready for this one.

Reply:
I can use some of these (space, borderWidth)... the colors are not correct for the borders though. Need to get these values from the SendGrid theme because the docs site uses the SG color palette.

Reply:
The site is using the SendGrid theme. I know @richbachman added the success and error borders. It might just need a small switch to the right aliases in the tokens file https://github.com/twilio-labs/paste/blob/master/packages/paste-design-tokens/tokens/themes/sendgrid/global/border-color.yml